File: WarehouseManagementWeb.Application/Dto/Input/ResourceShipment/ChangeStatusDocumentShipmentInput.cs
using WarehouseManagementWeb.Domain.Enums;

namespace WarehouseManagementWeb.Application.Dto.Input.ResourceShipment
{
    /// <summary>
    /// Класс входной модели обновления статуса документа отгрузки. 
    /// </summary>
    public class ChangeStatusDocumentShipmentInput
    {
        /// <summary>
        /// Id документа отгрузки.
        /// </summary>
        public int DocumentShipmentId { get; set; }

        /// <summary>
        /// Статус документа отгрузки.
        /// </summary>
        public DocumentStatusEnum StatusEnum { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/warehouse/document-shipment/models/input/change-status-document-shipment-input.ts
import { DocumentStatusEnum } from '../../../enums/document-status-enum';

/**
 * Класс входной модели обновления статуса документа отгрузки.
 */
export class ChangeStatusDocumentShipmentInput {
  /**
   * Id документа отгрузки.
   */
  documentShipmentId: number = 0;

  /**
   * Статус документа отгрузки.
   */
  documentShipmentStatusEnum: DocumentStatusEnum = 1;
}
